CMakeLists.txt

```diff
@@ -265,7 +270,7 @@ if(BUILD_TESTS)
     )
     use_client_mbedtls(kv_test)
     target_link_libraries(
-      kv_test PRIVATE ${CMAKE_THREAD_LIBS_INIT} secp256k1.host
+      kv_test PRIVATE ${CMAKE_THREAD_LIBS_INIT} secp256k1.host http_parser.host
```

Comment:
Why does `http_parser.host` need to be added here?

Reply:
`openapi.h` calls `http_method_str` to convert `http_method`s to strings. This comes from `http_parser`. So anything that includes `openapi.h` needs to link `http_parser` to get the implementation.